tools/packaging/packaging.psm1

```diff
@@ -826,8 +826,15 @@ function New-UnixPackage {
         }
```

Comment:
@TravisEz13, your last commit had 4 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/faq/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/faq/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```